### ISSUE TITLE
Cannot reproduce VC1-vitb on objnav?

### ISSUE THREAD

**Issue by jeasinema** [facebookresearch/eai-vc#13]
Thanks for this great project.

I followed the instructions of reproducing `objnav` results and use all the default hyper-parameters. Here is a glimp on validation set (`val_mini`):

![image](https://user-images.githubusercontent.com/10633528/236897499-1f866f6f-e997-4313-a5e0-f5ed0d3f07aa.png)

as you can see, `eval/success` is barely above zero. I believe the latest checkpoint I evaluated so far is from 6000th update step of training (the total update steps by default is [12000](https://github.com/facebookresearch/eai-vc/blob/e782c3551e288403f925f6022d37984349aa0102/cortexbench/habitat_vc/configs/experiments/objectnav_il.yaml#L28)) .

I'm wondering if this is normal? For your reference, here is the training curve:

![image](https://user-images.githubusercontent.com/10633528/236898205-74b9aaae-781a-421d-bf76-27c125209c9a.png)

(I think everything in training curve makes sense to me except for the success rate, it went on to 1 from the beginning)

**Comment by jeasinema** [comment]
Please note that I did use 1 node and 1 GPU only (this is done by commenting out the [slurm_run section](https://github.com/facebookresearch/eai-vc/blob/main/cortexbench/habitat_vc/configs/config_objectnav_il_frozen.yaml#L8)). But `NUM_ENVIRONMENTS` is still 16.

**Comment by xwen99** [comment]
They train with 512=16*32 environments in parallel (using 32 GPUs), so if you train with fewer GPUs, you may also need to set `NUM_ENVIRONMENTS` higher accordingly. In my setting I tried training on 8xA100s with 64 environments on each, and the curve looks fine.

**Comment by ykarmesh** [comment]
@xwen99's answer is correct. Number of envs and subsequently the batch size should be kept the same for all experiments.
